Add PR template

miguelemosreverte opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Ticket
No ticket

## Description
As you can see by this PR, the templating will provide a standard for pull requests.
This PR was created using the template.

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Refactor (non-breaking change which makes the codebase beautiful)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

miguelemosreverte committed:
Add PR template

miguelemosreverte commented:
@ffakenz Are we sure we are going to have the checklist or not?

Ready to remove the checklist on your behalf. 

Best regards, @miguelemosreverte

testagrossa commented:
remove the first 2 on the checklist
-  My code follows the code style of this project.
-  My change requires a change to the documentation.

miguelemosreverte committed:
removed the first 2 items on the PR template checklist:

My code follows the code style of this project.
My change requires a change to the documentation.
